wizard/src/utils/detect_date.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const MIN_HIGH_CONF_DETECT = 0.9;
+
+/**
+ * A DateDetector objected is meant to be initialized once. It provides
+ * convenience access to date detectors.
+ */
+export class DateDetector {
+  detectDate(header: string): boolean {
+    const d = Date.parse(header);
+    if (!Number.isNaN(d) && d >= 0) {
```

Comment:
nit: we can just do `return !Number.isNaN(d) && d >= 0;`

Reply:
Done, thanks!